### PR TITLE
Cleanup around S5693

### PR DESCRIPTION
**Pull request by costin-zaharia-sonarsource**

### CODE REVIEW

File: analyzers/tests/SonarAnalyzer.UnitTest/Rules/Hotspots/RequestsWithExcessiveLengthTest.cs [hunk 1]
```diff
@@ -34,56 +34,56 @@ namespace SonarAnalyzer.UnitTest.Rules
     [TestClass]
     public class RequestsWithExcessiveLengthTest
     {
+        private readonly VerifierBuilder builderCS = new VerifierBuilder()
+                                                     .AddAnalyzer(() => new CS.RequestsWithExcessiveLength(AnalyzerConfiguration.AlwaysEnabled))
+                                                     .WithBasePath(@"Hotspots")
+                                                     .AddReferences(GetAdditionalReferences());
```
Comment (mary-georgiou-sonarsource) [inline analyzers/tests/SonarAnalyzer.UnitTest/Rules/Hotspots/RequestsWithExcessiveLengthTest.cs:40]: nitpick:  Shouldn't the alignment be like this (not sure, but I'm a bit confused):
```suggestion    
{
        private readonly VerifierBuilder builderCS = new VerifierBuilder()
            .AddAnalyzer(() => new CS.RequestsWithExcessiveLength(AnalyzerConfiguration.AlwaysEnabled))
            .WithBasePath(@"Hotspots")
            .AddReferences(GetAdditionalReferences());
```

**Review by mary-georgiou-sonarsource — APPROVED** [review]
LGTM - nitpick comment on coding style.

File: analyzers/tests/SonarAnalyzer.UnitTest/Rules/Hotspots/RequestsWithExcessiveLengthTest.cs (hunk 1 above)
Comment (costin-zaharia-sonarsource) [reply analyzers/tests/SonarAnalyzer.UnitTest/Rules/Hotspots/RequestsWithExcessiveLengthTest.cs:40]: me neither :) 

for me, if we move the calls there they would be to far from the invocation target. I don't have a strong opinion about this.

Comment (mary-georgiou-sonarsource) [reply analyzers/tests/SonarAnalyzer.UnitTest/Rules/Hotspots/RequestsWithExcessiveLengthTest.cs:40]: I'll add an extra note in the discussion about chained invocations for this. :)

**Pull request merged**